### PR TITLE
idmocp-41: Append linters to the pipeline, and helper script for laun…

### PR DESCRIPTION
**Pull request by avisiedo**
- Append github actions for several linters.
- Append lint stage to the pipeline.
- Append helper script for launching linters locally.
- Update files with the linter hits.

### CODE REVIEW

File: deploy/crds/idmocp.redhat.com_idms_crd.yaml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+---
```
Comment (frasertweedale) [inline deploy/crds/idmocp.redhat.com_idms_crd.yaml:1]: This is an auto-generated file.  Therefore we should exclude it from the linter.

File: deploy/crds/idmocp.redhat.com_v1alpha1_idm_cr.yaml [hunk 2]
```diff
@@ -1,3 +1,4 @@
+---
```
Comment (frasertweedale) [inline deploy/crds/idmocp.redhat.com_v1alpha1_idm_cr.yaml:1]: This is another auto-generated file that should be excluded from the linter.

**Review by frasertweedale — CHANGES_REQUESTED** [review]
@avisiedo thanks for this mate!  There are a couple of files that are auto-generated by operator-sdk (and will be re-generated often as the operator evolves).  So those should be excluded from the linter.  I indicated which ones in comments.

File: deploy/crds/idmocp.redhat.com_idms_crd.yaml (hunk 1 above)
Comment (avisiedo) [reply deploy/crds/idmocp.redhat.com_idms_crd.yaml:1]: Thank you removing from the linter stage this file.

File: deploy/crds/idmocp.redhat.com_v1alpha1_idm_cr.yaml (hunk 2 above)
Comment (avisiedo) [reply deploy/crds/idmocp.redhat.com_v1alpha1_idm_cr.yaml:1]: Thank you removing from the linter stage this file.

**Comment by frasertweedale** [comment]
@avisiedo I'll review this tomorrow.

**Pull request merged**